docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -92,24 +75,15 @@ file called `policy.yaml`.
                         path: policy.yaml
 ----
 
-* You must preserve the value of the `host` option under the `[DEFAULT]`
-section as `hostgroup`.
-* The {rhos_component_storage_file} API service needs the `enabled_share_protocols` option to be
-added in the `customServiceConfig` section in `manila: template: manilaAPI`.
-* If you had scheduler overrides, add them to the `customServiceConfig`
+* The value of the `host` option under the `[DEFAULT]` section must be `hostgroup`.
+* To run the {rhos_component_storage_file} API service, you must add the `enabled_share_protocols` option to the `customServiceConfig` section in `manila: template: manilaAPI`.
+* If you have scheduler overrides, add them to the `customServiceConfig`
 section in `manila: template: manilaScheduler`.
-* If you had multiple storage backend drivers configured with {OpenStackShort} {rhos_prev_ver},
-you will need to split them up when deploying {rhos_acro} {rhos_curr_ver}. Each storage
-backend driver needs to use its own instance of the `manila-share`
-service.
-* If a storage backend driver needs a custom container image, find it on the
-https://catalog.redhat.com/software/containers/search?gs&q=manila[RHOSP Ecosystem Catalog]
-and create or modify an `OpenStackVersion` manifest to specify the custom image
-using the same `<custom name>`.
+* If you have multiple storage back-end drivers configured with {OpenStackShort} {rhos_prev_ver}, you need to split them up when deploying {rhos_acro} {rhos_curr_ver}. Each storage back-end driver needs to use its own instance of the `manila-share` service.
+* If a storage back-end driver needs a custom container image, find it in the
+link:https://catalog.redhat.com/software/containers/search?gs&q=manila[Red Hat Ecosystem Catalog], and create or modify an `OpenStackVersion` custom resource (CR) to specify the custom image using the same `custom name`.
```

Comment:
Is `custom_name` the actual name of the field in the the OpenStackVersion CR?

Reply:
@fmount Can I update this sentence to be a bit more specific? For example:
...and create or modify an `OpenStackVersion` custom resource (CR) to specify the custom image by using the same custom name in the `manilaShare` section.

Reply:
works for me!